### ISSUE TITLE
What should be the next step to compute point-to-point predictions after getting the (H/4, W/4) mask? Should it simply apply an interpolation on the (H/4, W/4) mask to obtain a prediction map of original size of (H, W)?

### ISSUE THREAD

**Issue by SudongCAI** [NVlabs/SegFormer#85]
As the above question.
Thank you so much!

**Comment by MorinCc** [comment]
I have the same question while I try to compute the loss that needs pixel-wise prediction.
And I wonder how the prediction is computed?

**Comment by pablogiaccaglia** [comment]
Yeah, before computing the loss. Something like
`pred = F.interpolate(pred, size = cfg.DATASET.CROP_SIZE, mode = 'bilinear', align_corners = False)` 
worked for me.
